packages/graphql/src/translate/queryAST/factory/FilterFactory.ts

```diff
@@ -809,8 +810,20 @@ export class FilterFactory {
         return this.wrapMultipleFiltersInLogical(filterTruthy(filterASTs));
     }
 
-    private createCountFilter(operatorKey: string, value: unknown, attachedTo: "node" | "relationship"): CountFilter {
+    private createCountFilter(
+        operatorKey: string,
+        value: unknown,
+        attachedTo: "node" | "relationship",
+        useDeprecated = true
+    ): CountFilter {
```

Comment:
With 4 parameters, considering changing this to an object input to give more context when calling it